### DISCUSSION TITLE
Calidad de caratulas descargadas

### DISCUSSION THREAD

**Discussion by Kendrick4K** [Paco8/plugin.video.movistarplus#61] [General]
Hola Paco.

Muchísimas gracias por este desarrollo. La app de movistar para Nvidia Shield no permite salida de UHD/4K ni sonido Dolby y con Kodi y tu addon si es posible. 

Quería consultarte por la calidad de las caratulas que se descargan de cada película o serie. No se si es problema de algún ajuste en Kodi o de tu software o que movistar las proporciona así. A mi particularmente se ven con baja calidad, no llegan a verse pixeladas pero si a muy baja resolución. ¿Hay que configurar algo para que se vean mejor?

Muchas gracias!

**Comment by Paco8** [comment]
En la versión 0.9.11 las carátulas tienen algo más resolución.
